src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -0,0 +1,38 @@
+package com.kodilla.ecommercee.domain;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table
```

Comment:
Jak nie zmieniamy nazwy tabeli to ta adnotacja nam nie jest potrzebna

Reply:
@metho Jeśli nie zmienię nazwy tabeli na "Orders" to wówczas otrzymuję błąd w trakcie uruchamiania aplikacji:

org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "create table order (order_id bigint not null auto_increment, number varchar(255) not null, order_date datetime not null, user_id bigint not null, primary key (order_id)) engine=MyISAM" via JDBC Statement

Reply:
Aplikacja zadziała, jeśli dodam do application.properties zapis:
spring.jpa.properties.hibernate.globally_quoted_identifiers=true

Mam dodać ten zapis, czy zmienić nazwę tabeli na "Orders"?